docs/changelog.rst

```diff
@@ -1,14 +1,14 @@
 Changelog
 ---------
 
-2022.8.0 (2022-08-18)
+2023.8.0
```

Comment:
why is 2022.8.0 being removed from the changelog?

Reply:
Oof, that was certainly not intentional ([b6cd09e](https://github.com/mozilla-services/python-dockerflow/pull/76/commits/b6cd09e6f7a0b9046318e3e2ad8f878ad0f0d272))